editor/editor_settings.cpp

```diff
@@ -998,6 +998,11 @@ const String EditorSettings::_get_project_metadata_path() const {
 	return EditorPaths::get_singleton()->get_project_settings_dir().path_join("project_metadata.cfg");
 }
 
+void EditorSettings::_remove_deprecated_settings() {
+	erase("run/output/always_open_output_on_play");
+	erase("run/output/always_close_output_on_stop");
+}
```

Comment:
This should be enclosed in `#ifndef DISABLE_DEPRECATED` (same for the declaration and the place where it's used).

Reply:
Done!
Did not know that even existed, thanks!